### DISCUSSION TITLE
[Feature] New Plug-in

### DISCUSSION THREAD

**Discussion by buiax** [Vinzent03/obsidian-git#439] [General]
Have you considered making a new plug-in to manage git repositories.

Get and display the Issues and pull-request from inside obsidian.

Similar to the JIRA plugin.

https://github.com/marc0l92/obsidian-jira-issue

**Comment by Vinzent03** [comment]
You are talking about a GitHub plugin right? Git itself doesn't have something like issues or pull requests.

What's your use case for this in Obsidian?
